public/locales/en-US/send.ftl

```diff
@@ -71,13 +70,15 @@ downloadFileSize = ({ $size })
 unlockInputLabel = Enter Password
 unlockInputPlaceholder = Password
 unlockButtonLabel = Unlock
-downloadFileTitle = Download Encrypted File
+downloadFilesTitle = Download Files
+downloadFileText = {$name} was encrypted with Firefox Send, and will expire after { $timeSpan } or { $downloadCount }.
```

Comment:
we don't reveal the expiry info to the downloader and the name is already on the next line